tools/data_manipulation/xarray/xarray_metadata_info.xml

```diff
@@ -1,41 +1,28 @@
-<tool id="netcdf_metadata_info" name="Netcdf Metadata Info" version="0.1.0">
+<tool id="xarray_metadata_info" name="Netcdf Metadata Info" version="0.15.1">
```

Comment:
if we change the ID or the TS repository name it is not the same tool anymore ... we get two tools in Galaxy.

Reply:
So maybe it was not smart to rename it? I thought it would be better to use xarray in the name but probably not.